### PR TITLE
Update dedicated host specs table

### PR DESCRIPTION
**Pull request by alexazl**
# Description
Update the storage column header in Dedicated Hosts specs table.

# Reasons
We want to be explicit that the spec refers to total storage, not available storage, in order to set the right expection with customers.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/dedicated-hosts-macos.adoc [hunk 1]
```diff
@@ -80,4 +80,4 @@ workflows:
 
 *A:* Apple released an https://www.apple.com/legal/sla/docs/macOSBigSur.pdf[updated end-user license agreement (EULA)] along with the release of Big Sur in November 2020, which requires cloud providers to lease Apple hardware to no more than one customer for a minimum of 24 hours.
 
-NOTE: By using the Services, you represent and warrant that you have reviewed and agree to be bound by the terms of the Apple Software License Agreement as well as the terms applicable to any software preinstalled on the Apple Software, including, but not limited to Apple's Xcode developer software and any other Apple or third-party software.
\ No newline at end of file
+NOTE: By using the Services, you represent and warrant that you have reviewed and agree to be bound by the terms of the Apple Software License Agreement as well as the terms applicable to any software preinstalled on the Apple Software, including, but not limited to Apple's Xcode developer software and any other Apple or third-party software.
```
Comment (thomasr8) [inline jekyll/_cci2/dedicated-hosts-macos.adoc:83]:
```suggestion
NOTE: By using the Services, you represent and warrant that you have reviewed and agree to be bound by the terms of the Apple Software License Agreement, as well as the terms applicable to any software preinstalled on the Apple Software, including, but not limited to, Apple's Xcode developer software and any other Apple or third-party software.
```

Comment (thomasr8) [reply jekyll/_cci2/dedicated-hosts-macos.adoc:83]: @alexazl Does this need to be reviewed by Legal or anyone? Just curious.

**Review by thomasr8 — COMMENTED** [review]
@alexazl Looks good, but you may wish to incorporate the minor suggestions, as well as reaching out to someone in Legal if we need to have them review the language.

File: jekyll/_cci2/dedicated-hosts-macos.adoc (hunk 1 above)
Comment (alexazl) [reply jekyll/_cci2/dedicated-hosts-macos.adoc]: @thomasr8 That language came from legal, so we're good to go there. Thanks for double-checking!

**Review by thomasr8 — APPROVED** [review]

**Pull request closed without merging**